### DISCUSSION TITLE
clarify calculation of max timeout when using RENOVATE_X_GITLAB_AUTO_MERGEABLE_CHECK_ATTEMPS

### DISCUSSION THREAD

**Discussion by sebhoss** [renovatebot/renovate#28983] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

### Please tell us more about your question or problem

https://docs.renovatebot.com/self-hosted-experimental/#renovate_x_gitlab_auto_mergeable_check_attemps mentions that the formula is `RENOVATE_X_GITLAB_MERGE_REQUEST_DELAY * attempt * attempt`. The same page mentions that `RENOVATE_X_GITLAB_MERGE_REQUEST_DELAY` is 250ms by default and `RENOVATE_X_GITLAB_AUTO_MERGEABLE_CHECK_ATTEMPS` is 5 by default. How does `250ms * 5 * 5` (= 6250ms = 6.25s) come out as `13.75s`?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
It backs off with each attempt. Search the source code to confirm

**Reply by sebhoss** [reply]
ok thanks

**Reply by HonkingGoose** [reply]
I think this is the code you want to read:

https://github.com/renovatebot/renovate/blob/da3872eda8e738cbc055cda1fc8d614c66272b80/lib/modules/platform/gitlab/index.ts#L647-L737
